domain/src/main/java/com/github/leuludyha/domain/model/library/Author.kt

```diff
@@ -23,4 +26,19 @@ data class Author(
     }
 
     override fun hashCode(): Int = id.hashCode()
+
+    fun toLoadedAuthor(): LoadedAuthor {
+        return runBlocking {
+            LoadedAuthor(
+                id = id,
+                name = name,
+                birthDate = birthDate,
+                deathDate = deathDate,
+                wikipedia = wikipedia,
+                works = works.first().map { it.toLoadedWork() },
+                covers = covers.first(),
```

Comment:
Have you tried what happens if the flow is empty when you call `.first()` ? is it safe?
If not consider calling `firstOrNull() ?: listOf()`